standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/properties/HMSServletTest.java

```diff
@@ -156,16 +169,19 @@ public Map<String, String> getProperties(List<String> selection) {
 
   @Test
   public void testServletEchoA() throws Exception {
-    URL url = new URL("http://hive@localhost:" + servletPort + "/" + path + "/" + NS);
+    String path = MetastoreConf.getVar(conf, MetastoreConf.ConfVars.PROPERTIES_SERVLET_PATH);
+    String scheme = MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.HTTPSERVER_USE_HTTPS)
+            ? "https" : "http";
```

Comment:
this is done like 3 times in this class itself, can we have a util like getScheme()